### DISCUSSION TITLE
How to use Trefle

### DISCUSSION THREAD

**Discussion by timcharlier6** [MDeLuise/plant-it#230] [Q&A]
Hi!
I found Plant-it on F-Droid and I'd like to use it for my garden. 
My problem is that I can only add plants manually, even after I put my trefle-key in server.env. 
I don't understand how it works. 
Take care

**Comment by MDeLuise** [comment]
Hi @timcharlier6,

Thank you for starting this discussion.

Unfortunately, the Trefle service is no longer operational. The project is inactive, and the dataset is no longer accessible. I’m currently working on [a fix](https://github.com/MDeLuise/plant-it/issues/218), but for now, it’s only possible to add plants manually.

I will keep you all updated on the progress. Thank you for your cooperation!
